recipe/meta.yaml

```diff
@@ -32,57 +22,28 @@ requirements:
     - {{ stdlib('c') }}
     - {{ compiler('cxx') }}
     - {{ compiler('rust') }}
+    - git
     - cmake
     - make
-    - libprotobuf
+    - pkg-config
   host:
     - python
     - pip
     - maturin >=1.4,<2.0
-    - libboost-devel
   run:
     - python
-    - numpy
-    - jinja2 >=2.0,<4
-    - ipywidgets >=7.5.1,<9
-    - pandas <3,>=0.22.0
-    - python-dateutil <3,>=2.8.0
-    - traitlets <6,>=4.3.2
 
 test:
   imports:
     - perspective
 
-outputs:
-  - name: perspective
-  - name: perspective-python
-    requirements:
-      build:
-        - {{ stdlib("c") }}
-      host:
-        - python
-      run:
-        - python
-        - perspective ={{ version }}
-
-  # TODO this is supposed to work, but fails with weird
-  # bugs. Try again in the future
-  # - name: perspective_python
-  #   build:
-  #     noarch: python
-  #   requirements:
-  #     host:
-  #       - python
-  #     run:
-  #       - python
-  #       - {{ pin_subpackage("perspective", exact=True) }}
-
 about:
   home: http://github.com/finos/perspective
   license: Apache-2.0
   license_family: Apache
   license_file:
```

Comment:
lets start to get better about including our statically linked licenses:
- https://github.com/conda-forge/polars-feedstock/blob/993ca7886232c75d5ab67715644b490bd04ae84f/recipe/meta.yaml#L39
- https://github.com/conda-forge/polars-feedstock/blob/993ca7886232c75d5ab67715644b490bd04ae84f/recipe/build.sh#L72
- https://github.com/conda-forge/polars-feedstock/blob/993ca7886232c75d5ab67715644b490bd04ae84f/recipe/meta.yaml#L66

Reply:
This doesn't seem relevant to this PR, how about you open an issue on the feedstock repo instead please

Reply:
This needs to be done for rust projects so I would consider it relevant

Reply:
https://github.com/conda-forge/perspective-feedstock/issues/182